editor/src/clj/editor/atlas.clj

```diff
@@ -40,6 +39,7 @@
             [editor.resource :as resource]
             [editor.resource-io :as resource-io]
             [editor.resource-node :as resource-node]
+            [editor.resource-dialog :as resource-dialog]
```

Comment:
Nit-pick: We try to keep these sorted by name, so this should technically be above `resource-node`.